test/base/test_env.py

```diff
@@ -11,14 +11,8 @@
 def test_vecenv(size=10, num=8, sleep=0.001):
     verbose = __name__ == '__main__'
     env_fns = [
-        lambda: MyTestEnv(size=size, sleep=sleep),
-        lambda: MyTestEnv(size=size + 1, sleep=sleep),
-        lambda: MyTestEnv(size=size + 2, sleep=sleep),
-        lambda: MyTestEnv(size=size + 3, sleep=sleep),
-        lambda: MyTestEnv(size=size + 4, sleep=sleep),
-        lambda: MyTestEnv(size=size + 5, sleep=sleep),
-        lambda: MyTestEnv(size=size + 6, sleep=sleep),
-        lambda: MyTestEnv(size=size + 7, sleep=sleep),
+        lambda: MyTestEnv(size=i, sleep=sleep)
+        for i in range(size, size + num)
```

Comment:
No, you can have a test and will find out that all env's size are identical.

Reply:
```python
In [17]: env_fns = [lambda: MyTestEnv(size=i, sleep=sleep) 
    ...:         for i in range(size, size + num)]                                                                                                                                                                  

In [18]: env_fn = [i() for i in env_fns]                                                                                                                                                                            

In [19]: print(env_fn[0].size, env_fn[1].size)                                                                                                                                                                      
17 17
```

Reply:
Yeh I found the issue. It was coming from the lambdas using current `i` value instead of the one existing at definition.